policy.md

```diff
@@ -1,30 +1,21 @@
 # Policy
 
-Olympus features policy constants that allow us to optimize the system. 
+Olympus features policy constants that allow us to optimize the system.
 
 ## Bonds
 
 The **BCV** allows us to scale the rate at which bond premiums increase. A higher BCV means a lower discount for bonders and less inflation. A lower BCV means a higher capacity for bonders and less protocol profit.
 
-The **vesting term** determines how long it takes for bonds to become redeemable. A longer term means lower inflation and lower bond demand. 
+The **vesting term** determines how long it takes for bonds to become redeemable. A longer term means lower inflation and lower bond demand.
```

Comment:
```suggestion
The **vesting term** determines how long it takes for bonds to become fully redeemable. A longer term means lower inflation and lower bond demand.
```

Reply:
Since bonds are vesting linearly, we can say they are _fully_ redeemable at the end of the vesting term.